.github/SUPPORT.md

```diff
@@ -1,28 +1,24 @@
-## Support for scikit-image
+## Technical support
 
 Welcome to scikit-image! We use GitHub for tracking bugs and feature requests.
 When you report an [issue](https://github.com/scikit-image/scikit-image/issues),
-you may apply a label to it to specify its type: bug, feature request, performance,
+go ahead and apply the appropriate label: bug, feature request, performance,
 infrastructure, documentation, etc.
```

Comment:
I don't think non-scikit-image members can actually apply labels themselves? E.g. I can't do that on repos I don't have the rights to change issues on.

I'll take the liberty to reword this bit.